modules/ballerina-core/src/main/java/org/ballerinalang/model/values/BJSON.java

```diff
@@ -296,7 +297,7 @@ public BIterator newIterator() {
                 }
             }
             if (arity == 1) {
-                return new BValue[]{new BJSON(collection.value.get((String) iterator.next()))};
+                return new BValue[]{new BJSON(iterator.next().getValue())};
```

Comment:
Please fix code formatting here. (how come this is not picked up by checkstyle?)